rules/ensure-that-the-scheduler-bind-address-argument-is-set-to-127.0.0.1/raw.rego

```diff
@@ -0,0 +1,55 @@
+package armo_builtins
+
+import future.keywords.in
+
+deny[msg] {
+	obj = input[_]
+	filter_input(obj)
+	result = invalid_flag(obj.spec.containers[0].command)
+
+	msg := {
+		"alertMessage": "the kube scheduler is not bound to a localhost interface only",
+		"alertScore": 2,
+		"failedPaths": result.failed_paths,
+		"fixPaths": result.fix_paths,
+		"packagename": "armo_builtins",
+		"alertObject": {"k8sApiObjects": [obj]},
+	}
+}
+
+filter_input(obj) {

```

Comment:
maybe let's give that filter more meaningful name like **is_kube_scheduler** ? 😁